src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = hidden;
+  }
+
+  hide() {
```

Comment:
The 'hide' method should only set the 'hidden' property to true. If there's a requirement for a method to toggle the 'hidden' state, it should be named accordingly to avoid confusion.